src/styles/product-card.scss

```diff
@@ -0,0 +1,80 @@
+.product-card {
+  border: 1px solid #fff;
```

Comment:
The border color is set to #fff, which is white. Ensure this is the intended design choice, as it might not be visible on a white background.